src/components/shared/Button/ThreeDotsButton.css

```diff
@@ -0,0 +1,42 @@
+.main {
```

Comment:
FYI Arren's recent pr also imports threedotsbutton, may be worth aligning

Reply:
Updated the colors so now its the exact same changes as [Migrate & Cleanup Colony Subscriptions]
#143